app/views/provider/admin/dashboards/_services_filter.html.slim

```diff
@@ -6,12 +6,13 @@
         var allApis = #{json @services}
 
         var displayApis = function(filteredApis) {
-          for (api = 0; api < allApis.length; api++) {
-            document.getElementById("service_" + allApis[api].service.id).style.display = "none"
-          }
-          for (api = 0; api < filteredApis.length; api++) {
-            document.getElementById("service_" + filteredApis[api].service.id).style.display = "inline"
-          }
+          allApis.forEach(function(api) {
```

Comment:
@thomasmaas browser support is ie9+, right?

Reply:
https://caniuse.com/#feat=es5 , looks pretty safe :)

Reply:
pretty much indeed!